deployment/src/main/java/io/quarkiverse/githubaction/deployment/GitHubActionProcessor.java

```diff
@@ -726,7 +721,9 @@ private static void generateMultiplexers(ClassOutput beanClassOutput,
                                 gitHubEventRh);
                     } else if (parameterAnnotations.stream().anyMatch(ai -> ai.name().equals(CONFIG_FILE))) {
                         AnnotationInstance configFileAnnotationInstance = parameterAnnotations.stream()
-                                .filter(ai -> ai.name().equals(CONFIG_FILE)).findFirst().get();
+                                .filter(ai -> ai.name().equals(CONFIG_FILE))
+                                .findFirst()
+                                .orElseThrow(() -> new AssertionError("ConfigFile annotation not present"));
```

Comment:
This actually can't happen given the check just above but I suppose it can't hurt.

Reply:
You're right, but it prevents yo from getting possibly really unhappy, if someone fiddles with the condition and forgets to change the handling of the optional. It's just a safety net.